drivers/pwm/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ zephyr_library_sources_ifdef(CONFIG_PWM_XLNX_AXI_TIMER	pwm_xlnx_axi_timer.c)
 zephyr_library_sources_ifdef(CONFIG_PWM_MCUX_PWT 	pwm_mcux_pwt.c)
 zephyr_library_sources_ifdef(CONFIG_PWM_GECKO   pwm_gecko.c)
 zephyr_library_sources_ifdef(CONFIG_PWM_GD32 pwm_gd32.c)
+zephyr_library_sources_ifdef(CONFIG_PWM_RPI_PICO	pwm_rpi_pico_slice.c)
```

Comment:
This can be `pwm_rpi_pico`. This has mostly got to do with the way the drivers is implemented right now.
It exposes a device for each individual pwm slice. I had some confusion regarding the pwm api. I'll rewrite
the driver to expose a single pwm device with 16 channels. I'll try to update this before the weekend.

Reply:
With the latest push this should be fixed